docs/sql-data-sources-csv.md

```diff
@@ -38,3 +38,222 @@ Spark SQL provides `spark.read().csv("file_name")` to read a file or directory o
 </div>
 
 </div>
+
+## Data Source Option
+
+Data source options of CSV can be set via:
+* the `.option`/`.options` methods of
+  *  `DataFrameReader`
+  *  `DataFrameWriter`
+  *  `DataStreamReader`
+  *  `DataStreamWriter`
+  *  `from_csv`
+  *  `to_csv`
+  *  `schema_of_csv`
+  *  `OPTIONS` clause at [CREATE TABLE USING DATA_SOURCE](sql-ref-syntax-ddl-create-table-datasource.html)
+
+
+<table class="table">
+  <tr><th><b>Property Name</b></th><th><b>Default</b></th><th><b>Meaning</b></th><th><b>Scope</b></th></tr>
+  <tr>
+    <td><code>sep</code></td>
+    <td>,</td>
+    <td>Sets a separator (one or more characters) for each field and value. If None is set, it uses the default value, <code>,</code>.</td>
```

Comment:
If `None` is set?

Reply:
Oh, seems like we don't need `If None is set ...` anymore. let me fix it. Thanks!